### DISCUSSION TITLE
📔【计算机网络】从输入一个 URL 到页面加载完成的过程

### DISCUSSION THREAD

**Discussion by imageslr** [imageslr/blog#78] [Announcements]
https://imageslr.github.io/2020/02/26/what-happens-when-you-type-in-a-url.html 

 前言 从输入一个 URL、按下回车到显示页面，中间发生了什么？这道题既有广度，又有深度，很能考验一个人的知识体系。

**Comment by fengxiaop** [comment]
HTTPS 本质上就是 HTTPS + SSL 应该是HTTP+SSL吧

**Comment by imageslr** [comment]
@fengxiaop 是一个错别字，谢谢指正
